community/samples/serving/helloworld-r/app.py

```diff
@@ -0,0 +1,15 @@
+import os
+import subprocess
+
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route('/')
+def hello_world():
+    output = subprocess.check_output('/usr/bin/Rscript HelloWorld.R', shell=True)
```

Comment:
If the Rscript errors out, will this respond with an HTTP 500?

Reply:
yes, but I updated to make it look nicer.